integration-kinesis-to-lambda-with-batch-item-handling/Function.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Text;
+using System.Text.Json.Serialization;
+using Amazon.Lambda.Core;
+using Amazon.Lambda.KinesisEvents;
+
+// Assembly attribute to enable the Lambda function's JSON input to be converted into a .NET class.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+
+namespace KinesisIntegration;
+
+public class Function
+{
+    public async Task<StreamsEventResponse> FunctionHandler(KinesisEvent evnt, ILambdaContext context)
+    {
+        if (evnt.Records.Count == 0)
+        {
+            context.Logger.LogLine("Empty Kinesis Event received");
```

Comment:
Could we update the sample to use PowerTools? Or at least mention it.

Reply:
updated code to add PowerTools logger